talentmap_api/fsbid/services/publishable_positions.py

```diff
@@ -236,3 +242,68 @@ def grade_map(x):
         'skillsFilters': list(map(skills_map, data.get('QRY_LSTSKILLCODES_DD_REF'))),
         'gradeFilters': list(map(grade_map, data.get('QRY_LSTGRADES_DD_REF'))),
     }
+
+def get_publishable_positions_csv(query, jwt_token, rl_cd, host=None):
+    args = {
+        "proc_name": 'qry_modPublishPos',
+        "package_name": 'PKG_WEBAPI_WRAP',
+        "request_mapping_function": publishable_positions_req_mapping,
+        "response_mapping_function": publishable_positions_res_mapping,
+        "jwt_token": jwt_token,
+        "request_body": query,
+    }
+
+    data = services.send_post_back_office(
+        **args
+    )
```

Comment:
```suggestion
    data = get_publishable_positions(query, jwt_token)
```

i think you may be able to do this

Reply:
yep, implemented